gitlab/__init__.py

```diff
@@ -68,7 +68,7 @@ class Gitlab(object):
 
     def __init__(
         self,
-        url,
+        url=DEFAULT_URL,
```

Comment:
@max-wittig can you think of this breaking anything? I couldn't think of a scenario :thinking: 

I was also thinking of adding the same for the CLI, but would like to add some cli args for the token for it to make sense, so I'll try in a follow-up.

Reply:
Oof, thanks for this, I found another bug based on your comment, as passing `None` explicitly as a positional arg with this would break the base url, so I pushed a new commit.

With this I think it covers all your scenarios as well (see new parametrized test in https://github.com/python-gitlab/python-gitlab/commit/77ac807f8c66d4debfde79ea450c4e35a17c506e) but I agree there may be other things. Should I mark this a draft and add it to a new 3.0.0 milestone? Is there any other breaking stuff maybe coming up other than the dream of httpx? :grin:

Reply:
I've added a breaking change trailer in the commit message to mark it for 3.0.0.